bioconda_utils/lint_functions.py

```diff
@@ -293,7 +293,7 @@ def pin(recipe, meta, df):
         # has jinja templating
         for line in open(os.path.join(recipe, 'meta.yaml')):
             line = line.strip()
-            if line.startswith('- {}'.format(dep_name)):
+            if line == '- {}'.format(dep_name):
```

Comment:
Wouldn't it be more robust to get the requirements from the `_get_deps()` function?